### ISSUE TITLE
Cant build using "Basic" example on Windows

### ISSUE THREAD

**Issue by slauzinho** [nozzle/react-static#195]
I tried running the basic example app choosing the basic example with:
`react-static create` 
```
? What should we name this project? my-static-site
? Select a template below... (Use arrow keys or type to search)
> basic
  apollo
  apollo-redux
  blank
  cordova
  custom-routing
  dynamic-imports
(Move up and down to reveal more choices)
```

And when I try to start it I get the following error:

```
Failed to rebuild.
./dist/react-static-routes.js
Syntax error: C:/XXXXX/dist/react-static-routes.js: Octal literal in strict mode (9:50)

   7 | import src_containers_Post from '..\src\containers\Post'
   8 | import src_containers_Blog from '..\src\containers\Blog'
>  9 | import src_containers_404 from '..\src\containers\404'
     |                                                   ^
  10 |     const templateMap = {
  11 |     t_0: src_containers_Home,
  12 | t_1: src_containers_About
```

**Comment by EmilTholin** [comment]
Hi @slauzinho! This has been [fixed in master](https://github.com/nozzle/react-static/pull/191). There will probably be a `4.3.2` release soon.

**Comment by tannerlinsley** [comment]
Should be fixed in 4.3.4 release! :)
